src/modules/App/stories/index.stories.js

```diff
@@ -269,6 +269,31 @@ export const Korean = () => fitPageSize(
   />
 );
 
+export const mobile = () => fitPageSize(
+  <App
+    appId={appId}
+    userId={array[0]}
+    nickname={array[0]}
+    profileUrl={addProfile}
+    breakpoint
+    showSearchIcon
+    allowProfileEdit
+    config={{ logLevel: 'all' }}
+    replyType="THREAD"
+    isReactionEnabled
+    isMentionEnabled
+    isVoiceMessageEnabled
+    isMultipleFilesMessageEnabled
+    isMessageReceiptStatusEnabledOnChannelList
+    uikitOptions={{
+      groupChannel: {
+        // enableTypingIndicator: false,
```

Comment:
Do we want to keep this comment too?

Reply:
Let's just remove it. I left it there so we can test below option when enableTypingIndicator is set to false